doc/whats_new.rst

```diff
@@ -144,6 +188,13 @@ Enhancements
    - The :func: `ignore_warnings` now accept a category argument to ignore only
      the warnings of a specified type. By `Thierry Guillemot`_.
 
+   - The new ``results_`` attribute of :class:`model_selection.GridSearchCV`
+     (and similar utilities) can be easily imported into pandas as a
+     ``DataFrame``. Ref :ref:`model_selection_changes` for more information.
+     (`#6697 <https://github.com/scikit-learn/scikit-learn/pull/6697>`_) by
```

Comment:
Okay, I don't mind; just seems like a lot of bookkeeping for whoever's doing the release if it's to be perfected.